shared/oval/sshd_use_approved_ciphers.xml

```diff
@@ -4,12 +4,12 @@
       <title>Use Only Approved Ciphers</title>
       <affected family="unix">
         <platform>multi_platform_rhel</platform>
+        <platform>multi_platform_debian</platform>
       </affected>
       <description>Limit the ciphers to those which are FIPS-approved and only
       use ciphers in counter (CTR) mode.</description>
     </metadata>
     <criteria operator="AND">
-      <extend_definition comment="Installed OS is certified" definition_ref="installed_OS_is_certified" />
```

Comment:
Do remember that FIPS 140-2 only applies for systems processing sensitive information.

If your SSH login is shared with other systems processing sensitive information, then SSH counts, however, this is *not* a blanket policy for all systems.

If this is a standalone system, processing no sensitive information, with independent login credentials, then FIPS 140-2 does not apply.

So, I guess that the two rule approach makes sense but, in the non-FIPS case, it just might not apply at all.

Technically, there should also be a check for applications that bring along their own OpenSSL/GnuTLS libraries and pretty much every Go and Rust binary on the system since none of those will be certified.